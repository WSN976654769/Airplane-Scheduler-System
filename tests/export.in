flight export flights1.csv
location export location1.csv
location import locations.csv
flight import flights6.csv
location import locations4.csv
location export location1s.csv
flight export flights1.csv
flight export
location export
exit