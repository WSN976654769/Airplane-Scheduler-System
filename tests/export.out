User: Exported 0 flights.

User: Exported 0 locations.

User: Imported 51 locations.

User: Imported 169 flights.
13 lines were invalid.

User: Imported 2 locations.

User: Exported 53 locations.

User: Exported 169 flights.

User: Error writing file.

User: Error writing file.

User: Application closed.
